inspirehep/base/templates/search/results.html

```diff
@@ -198,6 +220,28 @@
       </div>
     </div>
   </div>
+  {% with messages = get_flashed_messages(category_filter=["query_suggestion"]) %}
+    {% if messages %}
+    <div class="alert alert-warning">
+    <ul class=flashes>
+      {% for message in messages %}
+      <li class="fa fa-exclamation-triangle">   {{ message }}</li>
+      {% endfor %}
+    </ul>
+  </div>
+    {% endif %}
+    {% endwith %}
+    {% with messages = get_flashed_messages(category_filter=["query_suggestion_with_redirect"]) %}
+      {% if messages %}
+      <div class="alert alert-warning">
+      <ul class=flashes>
+        {% for message in messages %}
+        <li class="fa fa-exclamation-triangle">   {{ message }} Try: <a target="_blank" href='http://inspirehep.net/search?{{request.query_string}}'>inspirehep.net</a></li>
```

Comment:
Does `requests.query_string` need to be URL encoded? I guess Jinja will take care of escaping it for HTML but I am not sure it's correctly urlencoding it (e.g. spaces into %20 etc.)